e2e/actions/actions.go

```diff
@@ -1,1523 +1,1524 @@
-				sandbox,
-				"test", "-f", "/scratch/dir/file",
-			},
-			exit: 0,
-		},
-		{
-			name: "ScratchWorkdirBind",
-			args: []string{
-				"--workdir", hostWorkDir,
-				"--scratch", "/scratch",
-				"--bind", hostCanaryDir + ":/scratch/dir",
-				sandbox,
-				"test", "-f", "/scratch/dir/file",
-			},
-			postRun: checkHostDir(filepath.Join(hostWorkDir, "scratch/scratch", "dir")),
-			exit:    0,
-		},
-	}
-
-	for _, profile := range e2e.Profiles {
-		profile := profile
-		createWorkspaceDirs(t)
-
-		t.Run(profile.String(), func(t *testing.T) {
-			for _, tt := range tests {
-				c.env.RunSingularity(
-					t,
-					e2e.AsSubtest(tt.name),
-					e2e.WithProfile(profile),
-					e2e.WithCommand("exec"),
-					e2e.WithArgs(tt.args...),
-					e2e.PostRun(tt.postRun),
-					e2e.ExpectExit(tt.exit),
-				)
-			}
-		})
-	}
-}
-
-func (c actionTests) exitSignals(t *testing.T) {
-	e2e.EnsureImage(t, c.env)
-
-	tests := []struct {
-		name string
-		args []string
-		exit int
-	}{
-		{
-			name: "Exit0",
-			args: []string{c.env.ImagePath, "/bin/sh", "-c", "exit 0"},
-			exit: 0,
-		},
-		{
-			name: "Exit1",
-			args: []string{c.env.ImagePath, "/bin/sh", "-c", "exit 1"},
-			exit: 1,
-		},
-		{
-			name: "Exit134",
-			args: []string{c.env.ImagePath, "/bin/sh", "-c", "exit 134"},
-			exit: 134,
-		},
-		{
-			name: "SignalKill",
-			args: []string{c.env.ImagePath, "/bin/sh", "-c", "kill -KILL $$"},
-			exit: 137,
-		},
-		{
-			name: "SignalAbort",
-			args: []string{c.env.ImagePath, "/bin/sh", "-c", "kill -ABRT $$"},
-			exit: 134,
-		},
-	}
-
-	for _, tt := range tests {
-		c.env.RunSingularity(
-			t,
-			e2e.AsSubtest(tt.name),
-			e2e.WithProfile(e2e.UserProfile),
-			e2e.WithCommand("exec"),
-			e2e.WithArgs(tt.args...),
-			e2e.ExpectExit(tt.exit),
-		)
-	}
-}
-
-// E2ETests is the main func to trigger the test suite
-func E2ETests(env e2e.TestEnv) func(*testing.T) {
-	c := actionTests{
-		env: env,
-	}
-
-	return testhelper.TestRunner(map[string]func(*testing.T){
-		"action URI":            c.RunFromURI,          // action_URI
-		"exec":                  c.actionExec,          // singularity exec
-		"persistent overlay":    c.PersistentOverlay,   // Persistent Overlay
-		"run":                   c.actionRun,           // singularity run
-		"shell":                 c.actionShell,         // shell interaction
-		"STDPIPE":               c.STDPipe,             // stdin/stdout pipe
-		"action basic profiles": c.actionBasicProfiles, // run basic action under different profiles
-		"issue 4488":            c.issue4488,           // https://github.com/sylabs/singularity/issues/4488
-		"issue 4587":            c.issue4587,           // https://github.com/sylabs/singularity/issues/4587
-		"issue 4755":            c.issue4755,           // https://github.com/sylabs/singularity/issues/4755
-		"issue 4768":            c.issue4768,           // https://github.com/sylabs/singularity/issues/4768
+    // Copyright (c) 2019, Sylabs Inc. All rights reserved.
```

Comment:
Looks like the whole file has been indented

Reply:
Sorry - not sure what happened there...